complete/pom.xml

```diff
@@ -33,7 +33,7 @@
                                 <artifactItem>
                                     <groupId>org.apache.sling</groupId>
                                     <artifactId>org.apache.sling.installer.provider.installhook</artifactId>
-                                    <version>1.0.4</version>
```

Comment:
Look at https://github.com/Netcentric/accesscontroltool/issues/561. This does not work if you embed aecu.

Reply:
Yes, I mainly changed this to 1.1.0 as AEM cloud did not build the image with the old version.